### PR TITLE
Improvement: Added Infobox and Marketing button preset designs and SVG

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
Added Infobox and Marketing button preset designs and SVG
Removed advanced heading presets

### Types of changes
-breaking change which adds functionality

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by AbhijitNage123** [comment]
@supriyajjadhav Could you please check backword issue - https://share.getcloudapp.com/v1uLrBgY

**Comment by AbhijitNage123** [comment]
SVG mismatch issue Resolve. both info-box and marketing button preset's are working fine.

File: src/blocks/advanced-heading/settings.js [hunk 1]
```diff
@@ -530,6 +527,16 @@ const Settings = ( props ) => {
 			<InspectorControls>
 				<InspectorTabs>
 					<InspectorTab { ...UAGTabs.general }>
+						<UAGAdvancedPanelBody
+							title={ __( 'Presets', 'ultimate-addons-for-gutenberg' ) }
+							initialOpen={ true }
+						>
+							<UAGPresets
```
Comment (RajkiranBagal) [inline src/blocks/advanced-heading/settings.js]: Add this in a function

File: src/blocks/call-to-action/settings.js [hunk 2]
```diff
@@ -1144,6 +1143,16 @@ const Settings = ( props ) => {
 			<InspectorControls>
 				<InspectorTabs>
 					<InspectorTab { ...UAGTabs.general }>
+						<UAGAdvancedPanelBody
+							title={ __( 'Presets', 'ultimate-addons-for-gutenberg' ) }
+							initialOpen={ true }
+						>
```
Comment (RajkiranBagal) [inline src/blocks/call-to-action/settings.js]: Add this in a function

File: src/blocks/faq/settings.js [hunk 3]
```diff
@@ -1058,6 +1057,16 @@ const Settings = ( props ) => {
 			<InspectorControls>
 				<InspectorTabs>
 					<InspectorTab { ...UAGTabs.general }>
+						<UAGAdvancedPanelBody
+							title={ __( 'Presets', 'ultimate-addons-for-gutenberg' ) }
+							initialOpen={ true }
+						>
```
Comment (RajkiranBagal) [inline src/blocks/faq/settings.js]: Add this in a function & all like these

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**